### ISSUE TITLE
Examples

### ISSUE THREAD

**Issue by quico14** [subor/nng.NETCore#50]
Hi guys,

I am trying to repeat your Publisher/Subscriber example, but with the test examples is not enough to get how to properly do it. Then, could you provide any example (no test), please?

Thanks.

**Comment by jake-ruyi** [comment]
Yep, after you opened #47 I vowed to put together two stand-alone examples this weekend:  
1. Using the "low-level" native NNG API
1. Using the higher-level wrapper

Should have it checked in later today.

**Comment by zhoujin7** [comment]
Sorry, I don't know how to use it in .Net Framework 4.6.1+.
